### DISCUSSION TITLE
Feature request - Use schemas for request bodies

### DISCUSSION THREAD

**Discussion by manuel-watchenterprise** [dedoc/scramble#359] [Ideas]
@romalytvynenko  Is there any plan to extract request bodies into schemas?

This would not only help to reuse some requests instead of multiplying them but also almost essential for generator(s). At least all object types in requests need `title` attribute, but the best would be if they were extracted to schemas.

I know it's hard to maintain and easy to request everything 😀

**Comment by romalytvynenko** [comment]
@manuel-watchenterprise I think this is a great idea and definitely makes sense. Feel free to create a PR for this.
